### PR TITLE
Updated readme contributors section

### PR DESCRIPTION
**Pull request by eliasjpr**
- Added new maintainer Dru Jensen - Welcome! The team excited to have you! :)
- Remove creator from contributors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,10 +155,10 @@ html
 
 ## Contributors
 
-- [eliasjpr](https://github.com/eliasjpr) Elias Perez - creator, maintainer
+- [eliasjpr](https://github.com/eliasjpr) Elias Perez - Maintainer
 - [fridgerator](https://github.com/fridgerator) Nick Franken - Maintainer
 - [elorest](https://github.com/elorest) Isaac Sloan - Maintainer
-- [phoffer](https://github.com/phoffer) Paul Hoffer - Maintainer
+- [drujensen](https://github.com/drujensen) Dru Jensen - Maintainer
 - [bew](https://github.com/fridgerator) Benoit de Chezelles - Member
```
Comment (bew) [inline README.md]: Just to mention, it's not the correct url here

Comment (eliasjpr) [inline README.md]: 👍🏻 I'll ask him

Comment (elorest) [inline README.md]: What he's saying is that instead of his email it's just fridgerators again.

Comment (eliasjpr) [inline README.md]: Ahh! Sorry @bew I read this from my phone I will correct your github tag my apologies my friend

**Pull request merged**